consyncful.gemspec

```diff
@@ -41,7 +41,7 @@ Gem::Specification.new do |spec|
 
   spec.add_dependency 'contentful', ['>=2.11.1', '<3.0.0']
   spec.add_dependency 'hooks', '>=0.4.1'
-  spec.add_dependency 'mongoid', ['9.0.0']
+  spec.add_dependency 'mongoid', '~> 9'
```

Comment:
I am wondering if we should open this up to a future version, ie `>= 9`

I could go either way. Seems silly to have to do this manually, on the other hand maybe its good to make sure the tests pass on a appropriate version of mongo.